webview-ui-vite/src/components/ChatView/ButtonSection.tsx

```diff
@@ -46,10 +43,10 @@ const ButtonSection: React.FC<ButtonSectionProps> = ({
 					opacity: primaryButtonText || secondaryButtonText ? (enableButtons ? 1 : 0.5) : 0,
 					display: "flex",
 				}}>
-				{primaryButtonText && (
+				{!isRequestRunning && primaryButtonText && (
 					<VSCodeButton
 						appearance="primary"
-						disabled={!enableButtons || isRequestRunning}
+						disabled={isRequestRunning}
```

Comment:
i think we should keep the `!enableButtons`

Reply:
agree